var/spack/repos/builtin/packages/r-deseq2/package.py

```diff
@@ -34,9 +34,10 @@ class RDeseq2(RPackage):
     homepage = "https://www.bioconductor.org/packages/DESeq2/"
     url      = "https://git.bioconductor.org/packages/DESeq2"
 
+    version('1.18.1', git='https://git.bioconductor.org/packages/DESeq2', commit='ef65091d46436af68915124b752f5e1cc55e93a7')
     version('1.16.1', git='https://git.bioconductor.org/packages/DESeq2', commit='0a815574382704a08ef8b906eceb0296f81cded5')
 
-    depends_on('r@3.4.0:3.4.9', when='@1.16.1')
+    depends_on('r@3.4.0:3.4.9', when='@1.16.1:')
```

Comment:
are you sure that `1.18.X` doesn't work with newer versions of `r`?

Reply:
If you mean R 3.5.x I don't know (this version of deseq2 was released last december). I've tested it only with R 3.4.x.